rtv/curses_helpers.py

```diff
@@ -183,9 +185,77 @@ def init(cls):
         # Assign the terminal's default (background) color to code -1
         curses.use_default_colors()
 
+        # Set default colors
+        cls._colors.update( config.default_colors )
+        # Override default colors by theme colors
+        cls._colors.update( cls.load_colors(config.theme) )
+
         for index, (attr, code) in enumerate(cls._colors.items(), start=1):
             curses.init_pair(index, code[0], code[1])
-            setattr(cls, attr, curses.color_pair(index))
+            color = curses.color_pair(index)
+            # add format to color
+            for f in code[2:4]: color |= f
+            setattr(cls, attr, color)
+
+    @classmethod
+    def load_colors(cls, theme):
+        # get all available themes
+        themes = cls.load_themes()
+        # get theme to use if exists
+        theme  = themes[theme] if theme in themes.keys() else {}
+
+        # compute colors hash
+        colors = {}
+        # loop in theme entries
+        for key in theme:
+            values = []
+            for value in theme[ key ].strip().split(','):
+                # check for aliases
+                if value in config.color_aliases.keys():
+                    values.append( config.color_aliases[ value ] )
+                else:
+                    # integer is required
+                    try:
+                        values.append( int( value ) )
+                    except:
+                        pass
+
+            colors[ key ] = values
+        return colors
+
+    @classmethod
+    def load_themes(cls):
+        """ Load all themes in config paths, return theme indexed hash of colors  """
```

Comment:
My idea was to allow us to define several themes in one file.

All files matching with `<RTV_CONFIG_PATH>/colors/*.cfg` are read because the theme id (example: `monokai-extended`) do not necessarily correspond to file name.

The goal is allow to create themes like:
```
[themename]
...

[themename-dark]
...

[themename-light]
...
```
This theme file will be surely name `themename.cfg`, but theme defined in `rtv.cfg` may be `themename-dark`

I don't know if it is a required feature, maybe we just should delete it.

Currently I read all themes and don't break the loop, at least I can break the loop when theme is found.